apps/crawler-fetcher/tests/python/test_podcast_resolve_url.py

```diff
@@ -35,18 +34,21 @@ def test_get_feed_url_from_google_podcasts_url():
     assert _get_feed_url_from_google_podcasts_url('http://www.example.com/') == 'http://www.example.com/'
     assert _get_feed_url_from_google_podcasts_url('totally not an URL') == 'totally not an URL'
 
-    ra_feed_url = 'https://www.residentadvisor.net/xml/podcast.xml'
+    npr_feed_url = 'https://feeds.npr.org/381444908/podcast.xml'
```

Comment:
RA seems not to be a registered publisher on Google Podcasts anymore or something, so the formatting of their page is weird; switched to NPR instead for that reason

<img width="686" alt="Screen Shot 2021-01-25 at 8 22 59 PM" src="https://user-images.githubusercontent.com/6501236/105782627-433a1d00-5f4b-11eb-8107-56b0433f8d79.png">